docs/whats-new.md

```diff
@@ -2,30 +2,55 @@
 
 This page contains highlights of each deck.gl release. Also check our [vis.gl blog](https://medium.com/vis-gl) for news about new releases and features in deck.gl.
 
-## deck.gl v9.0.0-beta (In Development)
+## deck.gl v9.0.0
 
 deck.gl 9.0 development is actively ongoing with the goal of releasing a beta version as soon as possible.
 
-Target release date: end of January 2024
+Release date: March 15, 2024
+
+<table style={{border: 0}} align="center">
+  <tbody>
+    <tr>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/collision-filter-extension.gif?raw=true" />
+        <p><i>WebGPU ready</i></p>
+      </td>
+      <td>
+        <img style={{maxHeight:200}} src="https://github.com/visgl/deck.gl-data/blob/master/images/whats-new/terrain-extension.gif?raw=true" />
```

Comment:
I'm aware that the highlighted features are things that I've worked on & had high visibility of. Happy to replace with other things, I just think it is nice to have this visual section at the top